MOLLER.yaml

```diff
@@ -0,0 +1,119 @@
+# Name is the short name of the VO
+Name: MOLLER
+# LongName is the full name of the VO
+LongName: The Measurement of a Lepton-Lepton Electroweak Reaction
+# AppDescription is a brief description of the VO
+AppDescription: MOLLER Simulation and Data Analysis
+# Community is the description of the community the VO belongs to
+Community: The MOLLER Collaboration
+
+CertificateOnly: false
+
+# If you have an up-to-date local git clone, fill ID with the output from `bin/next_virtual_organization_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+ID: <ID>
+
+# Contacts contain information about people to contact regarding this VO.
+# The "ID" is a hash of their email address available at https://topology.opensciencegrid.org/miscuser/xml
+# If you cannot find the contact in the above XML, please register the contact:
+# https://opensciencegrid.org/docs/common/registration/#registering-contacts
+Contacts:
+  # Administrative Contact is a list of people to contact regarding administrative issues
+  Administrative Contact:
+  - ID: <EMAIL HASH>
+    Name: <FULL NAME>
+  # - ...
+
+  # Security Contact is a list of people to contact regarding security issues
+  Security Contact:
+  - ID: <EMAIL HASH>
+    Name: <FULL NAME>
+  # - ...
```

Comment:
We'll need registered contacts for both of these fields. Could you have those contacts register through these instructions?

Reply:
Hi @rahmans1, I 've approved your registration so we should be able to fill these fields in for you. However, it would be ideal if we would two or more contacts for both the administrative + security fields. Is there anyone else from MOLLER that would be able to register as a contact?